packages/api-symphony-compatibility/src/mapping.ts

```diff
@@ -0,0 +1,56 @@
+const containerjsSsf = ssf;
+
+namespace map {
+  export class ssf {
+    static activate(windowName: string) {
+      // Could use window.focus in containerjs, but no way of selecting window by name yet
+      return;
+    }
+
+    static registerActivityDetection(throttle: number, callback: Function) {
```

Comment:
For these methods I think it would be useful to reference the symphony confluence URL where they are specifed